src/base/torrentfilter.cpp

```diff
@@ -34,6 +34,7 @@
 const std::optional<QString> TorrentFilter::AnyCategory;
 const std::optional<TorrentIDSet> TorrentFilter::AnyID;
 const std::optional<Tag> TorrentFilter::AnyTag;
+const std::optional<bool> TorrentFilter::AnyIsPrivate;
```

Comment:
I would name it `AnyPrivate`.

Reply:
`AnyPrivate` looks confusing.

Now since we use `std::optional` these constants don't do much good. We can use `std::nullopt` directly and not bother to come up with names for these constants.

Reply:
>We can use std::nullopt directly and not bother to come up with names for these constants.

👍